examples/training/classification/basic_imagenet_training.py

```diff
@@ -47,22 +47,32 @@
 
 """
 
+flags.DEFINE_string(
+    "model_name", None, "The name of the model in KerasCV.models to use."
+)
 flags.DEFINE_string("imagenet_path", None, "Directory from which to load Imagenet.")
 flags.DEFINE_string(
-    "backup_path", None, "Directory which will be used for training backups"
+    "backup_path", None, "Directory which will be used for training backups."
 )
 flags.DEFINE_string(
     "weights_path", None, "Directory which will be used to store weight checkpoints."
 )
 flags.DEFINE_string(
     "tensorboard_path", None, "Directory which will be used to store tensorboard logs."
 )
+flags.DEFINE_integer("batch_size", 256, "Batch size for training and evaluation.")
```

Comment:
I'm not a huge fan of "basic_imagenet_training.py"

can we use the structured `examples/training/classification/imagenet/basic_training.py`, include a README providing an overview of the task, say we use .tfrecord loading, etc.

Then in subdirectories we can have specific scripts: "ResNet50"

Reply:
basic_imagenet_training.py is a good name for a keras.io tutorial - but we can do that in keras.io when we move there.

Reply:
Done -- restructured again as discussed